inginious/frontend/pages/admin/admin.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" Admin index page"""
+
+from flask import request, jsonify
+
+from inginious.frontend.pages.utils import INGIniousAdministratorPage
+
+
+class AdministrationUsersPage(INGIniousAdministratorPage):
+    """User Admin page."""
+
+    def GET_AUTH(self, *args, **kwargs):
+        """ Display admin users page """
+        return self.show_page()
+
+    def POST_AUTH(self, *args, **kwargs):
+        """ Display admin users page"""
+        return self.show_page()
+
+    def show_page(self):
+        """Display page"""
+
+        page = int(request.form.get("page")) if request.form.get("page") is not None else 1
+        user_per_page = 10  # TODO probably better to let user define user_per_page
+        all_users, size_users = self.user_manager.get_users(limit=user_per_page, skip=(page-1)*user_per_page)
+        activated_users = self.user_manager.get_activated_users(list(all_users))
```

Comment:
This acts like a filter on previous results but still make new requests. It's probabl worth adding an ``activated`` field to the ``UserInfo`` class in order to avoid this inefficient behaviour.